src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -590,6 +590,8 @@ winrt::com_ptr<CascadiaSettings> CascadiaSettings::FromJson(const Json::Value& j
 // <none>
 void CascadiaSettings::LayerJson(const Json::Value& json)
 {
+    // add a new inheritance layer, and apply json values to child
+    _globals.attach(_globals->CreateChild());

```

Comment:
note that your inheritance here is incomplete! the baby globals has no color schemes, because they're not inherited... and if they were, you'd want it to be a .. copy? it's hard to have a layered array, for sure.